### ISSUE TITLE
Can't generate project

### ISSUE THREAD

**Issue by Timopheym** [shovon/generator-react-material-ui#1]
``` bash
Timopheys-MacBook-Pro:test timopheym$ yo react-material-ui
? The title of your project: SomeName
   create src/index.html
   create src/app.jsx

/usr/local/lib/node_modules/generator-react-material-ui/node_modules/yeoman-generator/lib/base.js:769
    self.conflicter.checkForCollision(file.path, file.contents.toString(), fun
                                                               ^
TypeError: Cannot call method 'toString' of null
    at DestroyableTransform._transform (/usr/local/lib/node_modules/generator-react-material-ui/node_modules/yeoman-generator/lib/base.js:769:64)
    at DestroyableTransform.Transform._read (/usr/local/lib/node_modules/generator-react-material-ui/node_modules/yeoman-generator/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (/usr/local/lib/node_modules/generator-react-material-ui/node_modules/yeoman-generator/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/usr/local/lib/node_modules/generator-react-material-ui/node_modules/yeoman-generator/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at writeOrBuffer (/usr/local/lib/node_modules/generator-re  


Timopheys-MacBook-Pro:gen timopheym$ yo -v
1.3.3
Timopheys-MacBook-Pro:gen timopheym$ node -v
v0.10.29
Timopheys-MacBook-Pro:gen timopheym$ npm -v
1.4.9
Timopheys-MacBook-Pro:gen timopheym$ uname -a
Darwin Timopheys-MacBook-Pro.local 14.0.0 Darwin Kernel Version 14.0.0: Fri Sep 19 00:26:44 PDT 2014; root:xnu-2782.1.97~2/RELEASE_X86_64 x86_64
```

**Comment by alexortiz201** [comment]
I second this

**Comment by shovon** [comment]
I'm sorry if my comment isn't really useful, but I'm currently in the process of re-writing a lot of the code. 

Please take a look at the [bankruptcy](https://github.com/shovon/generator-react-material-ui/tree/bankruptcy) branch.

(I'm calling it "bankruptcy" in reference to [Reginald Braitwaite](http://braythwayt.com/)'s [tweet](https://twitter.com/raganwald/status/551800056207003648) about how a rewrite is like declaring bankruptcy on technical debt. I was planning on adding tests for a while, but never did. It became so bad that it was a total pain to add new features. Hence, I'm rewriting the whole--or at least most of the--codebase to add tests.)

**Comment by shovon** [comment]
Let's see if commit 58209a174f457bcba6baa7ac9f8991da412b0704 fixes it.

It's now up on npm, as version 0.2.0.

**Comment by shovon** [comment]
Made another change. I think the problem arose when Yeoman tried looking for a `.gitignore` but wasn't able to find it, because npm purges it before publishing it out into the wild.

bbfc4ff0be0be40c7700173999d65162977c4163

**Comment by shovon** [comment]
So yeah, check out version 0.2.1, and see what you get.

**Comment by shovon** [comment]
Yet another bump to the version number. Please checkout 0.3.0.

**Comment by jarvismartin** [comment]
I ran 'yo react-material-ui projectname', and it seems to have worked just fine. However, I can't scaffold a page. I get the same error whether I provide a page name or not.

```
$ yo react-material-ui:page issues

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Did not provide required argument pagetitle!
```

**Comment by jarvismartin** [comment]
I ran 'yo react-material-ui <projectname>', and it seems to have worked just fine. However, I can't scaffold a page. I get the same error whether I provide a page name or not.

    $ yo react-material-ui:page issues

    events.js:72
            throw er; // Unhandled 'error' event
                  ^
    Error: Did not provide required argument pagetitle!

**Comment by shovon** [comment]
Have you tried providing a second argument? e.g.

``` shell
$ yo react-material-ui:page issues Issues
```

(Maybe I should remove the requirement for an explicit title)

**Comment by jarvismartin** [comment]
Yes, that works! Thanks.

**Comment by lukefrake** [comment]
I found this issue after failing on the README.md. :+1: to removing the explicit title or update the README

**Comment by shovon** [comment]
I'm not locking this issue, but it seems that it has been resolved back in bbfc4ff0be0be40c7700173999d65162977c4163, and was pushed to npm as version 0.2.1.

Closing until further notice.
